### DISCUSSION TITLE
Difficulties to setup display

### DISCUSSION THREAD

**Discussion by Dalotel** [VirusPilot/stratux-pi4#7] [Q&A]
Bonjour,
I am in the process to setup my PI4. I am just at the end of the rasbian installation.
I have a problem how to do with this sentence: **in /etc/rc.local you need to enable your particular display type after the setup script has finished**
I am totally beginner with rasp and others relative things.
I connect a OLED 1.5inch directly to my PI4 and yesterday I follow the entire process. But the display didn't work.
Today I format my SD card and redo the entire process.

Thanks for your help

Armel

**Comment by VirusPilot** [comment]
When connected via ssh as root, just uncomment the following line in /etc/rc.local so that it reads:

(sleep 60; python3 /root/stratux-radar-display/main/radar.py -d Oled_1in5 -c $_IP) &

In some cases -c $_IP won't work, then just use:

(sleep 60; python3 /root/stratux-radar-display/main/radar.py -d Oled_1in5 -c 192.168.10.1) &

or

(sleep 60; python3 /root/stratux-radar-display/main/radar.py -d Oled_1in5 -c 192.168.1.1) &

Depending on your Stratux settings.
